### ISSUE TITLE
Uncorrect impact of inversion

### ISSUE THREAD

**Issue by Artemlin6231** [Ensembl/ensembl-vep#1516]
## Describe the issue
We investigated how vep annotates inversions and performed an inversion in the vcf file on the region chr17:7671525-7685229, where vep was expected to assign a HIGH impact to this inversion since it encompasses the exons of the TP53 gene. However, vep assigned it a MODIFIER status.

### System
- VEP version: 110

### Full VEP command line
```
vep -i /mnt/hetero.vcf --gff /mnt_hg38/hg38.annotation.gff3.gz --fasta /mnt_hg38/hg38.fa -o hetero -q --tab --symbol --max_sv_size 1000000
```

**Comment by Artemlin6231** [comment]
input file contained the line chr17   7671524 2427    N       <INV>   371.6   .       SVTYPE=INV;SVLEN=13705;END=7685229;STRANDS=++:1,--:10;IMPRECISE;CIPOS=-27,27;CIEND=-246,29;CIPOS95=-10,5;CIEND95=-94,1;SU=29 ...

**Comment by likhitha-surapaneni** [comment]
Hi @Artemlin6231 , thank you for providing us with the input file. VEP currently assigns IMPACT as per this [table](http://www.ensembl.org/info/genome/variation/prediction/predicted_data.html). Could you please elaborate on the expected output and which transcripts should instead have a HIGH impact?

**Comment by Artemlin6231** [comment]
Hi! 
I expected to see a HIGH impact because the inversion intersects with several exons of the functionally significant TP53 gene. I am attaching a screenshot indicating the location of this inversion, and a file where I annotated the inversions using VEP

<img width="1039" alt="image_2024-03-22_17-45-04" src="https://github.com/Ensembl/ensembl-vep/assets/71922548/43ed1c7a-d9b0-4099-bd23-14f5d5c28b20">
[output_invers.txt](https://github.com/Ensembl/ensembl-vep/files/14725304/output_invers.txt)

**Comment by likhitha-surapaneni** [comment]
Hi @Artemlin6231 ,

Thank you for your detailed response. Following feedback from multiple users, we are planning to reclassify inversion as high impact in the upcoming releases.
I am closing this ticket for now. Please feel free to open a new issue if there are any other questions.

Kind regards,
Likhitha

**Comment by jamie-m-a** [comment]
Hi @Artemlin6231 

I have a quick update on this issue. the problem is not the inversion impact. Impact is set according to consequence rather than variant type. In the example you've given there is something going wrong with our consequence calculation for this inversion. It's currently coming up as intronic, but it clearly impacts the coding region as you've said. We'll continue to investigate and look to get a fix in place. I'll update again once it's fully resolved.

**Comment by jamie-m-a** [comment]
Actually the ALT allele is missing from the example you gave, when you add it in the consequence is more appropriate:

`chr17 7671524 2427 N 371.6 . SVTYPE=INV;SVLEN=13705;END=7685229;STRANDS=++:1,--:10;IMPRECISE;CIPOS=-27,27;CIEND=-246,29;CIPOS95=-10,5;CIEND95=-94,1;SU=29`

<img width="1270" alt="Image" src="https://github.com/user-attachments/assets/278589b1-e77c-4553-8120-2202e16ba6dd" />

The impact is still modifier as impact is a very broad term and we generally don't recommend that too much weight is given to it when considering the significance of a variant, instead the predicted consequence of the variant along with your own knowledge of the affected gene are more important when assessing the potential relevance and importance of the variant.

In summary, Ensembl VEP is behaving as expected for this variant, once the input is corrected to meet the VCF specification for an inversion.

**Comment by likhitha-surapaneni** [comment]
Hi, I am closing this ticket since the issue is addressed. Please feel free to open a new ticket in case of other issues
